pkg/services/accesscontrol/database/resource_permissions.go

```diff
@@ -214,13 +298,15 @@ func (s *AccessControlStore) createResourcePermission(sess *sqlstore.DBSession,
 		tr.team_id AS team_id,
 		t.name AS team,
 		t.email AS team_email,
-		r.name as role_name
+		r.name as role_name,
+		br.role as built_in_role
 	FROM permission p
 		LEFT JOIN role r ON p.role_id = r.id
 		LEFT JOIN team_role tr ON r.id = tr.role_id
 		LEFT JOIN team t ON tr.team_id = t.id
 		LEFT JOIN user_role ur ON r.id = ur.role_id
 		LEFT JOIN ` + s.sql.Dialect.Quote("user") + ` u ON ur.user_id = u.id
+		LEFT JOIN builtin_role br ON r.id = br.role_id
```

Comment:
nit: could we reuse `getResourcePermissions` to find the permission that has just been inserted? Or at least pull the `rawSql` out so that it can be reused?

Reply:
Good idea!